cfme/intelligence/reports/widgets/__init__.py

```diff
@@ -100,6 +94,44 @@ def delete(self, cancel=False):
             view.flash.assert_no_error()
 
 
+class DashboardReportWidgetsCollection(BaseCollection):
+    ENTITY = BaseDashboardReportWidget
+
+    @property
+    def CHART(self):  # noqa
+        from cfme.intelligence.reports.widgets.chart_widgets import ChartWidget
+        return ChartWidget
+
+    @property
+    def MENU(self):  # noqa
+        from cfme.intelligence.reports.widgets.menu_widgets import MenuWidget
+        return MenuWidget
+
+    @property
+    def RSS(self):  # noqa
+        from cfme.intelligence.reports.widgets.rss_widgets import RSSFeedWidget
+        return RSSFeedWidget
+
+    @property
+    def REPORT(self):  # noqa
+        from cfme.intelligence.reports.widgets.report_widgets import ReportWidget
+        return ReportWidget
+
+    def instantiate(self, widget_class, *args, **kwargs):
+        return widget_class.from_collection(self, *args, **kwargs)
+
+    def create(self, widget_class, *args, **kwargs):
+        """Create this Widget in the UI."""
+        dashboard_widget = self.instantiate(widget_class, *args, **kwargs)
+        view = navigate_to(dashboard_widget, "Add")
+        view.fill_with(dashboard_widget.fill_dict, on_change=view.add_button.click)
+        view = dashboard_widget.create_view(AllDashboardWidgetsView)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+        view.flash.assert_message('Widget "{}" was saved'.format(dashboard_widget.title))
```

Comment:
**Optional**

Move this flash content validation to the test_crud methods you're updating below, and only `assert_no_error` in this create() method.